Methods.md

```diff
@@ -1,39 +1,49 @@
 ## 1 Methods
 
 ### 1.0 Naming
+
 Method names should be a summary of what the method is doing, it needs to stay percise and short and representative of the operation with respect to synchrony.
 
 #### 1.0.0 Verbs
+
 Method names must contain verbs in them to represent the action it performs.
+
 ##### Do
+
 ```cs
 public List<Student> GetStudents()
 {
-	...
+    ...
 }
-
 ```
+
 ##### Don't
+
 ```cs
-public List<Student> Students(){
```

Comment:
I put the brace in new line, it distracts the reader from the original difference between the snippets.